dev/bin/kafka-setup.sh

```diff
@@ -1,126 +0,0 @@
-#!/bin/bash
```

Comment:
deletion of this file, breaks `kustomize/startMinikubeDeployLocalDev.sh`

especially:
```
echo "Setup Managed Kafka and Managed Connectors"
. ${BIN_DIR}/configure.sh kafka managed-connectors
${BIN_DIR}/kafka-setup.sh
```

Is there a way to fix it please?

Reply:
done